tfhe/src/shortint/ciphertext/zk.rs

```diff
@@ -141,6 +146,74 @@ impl ProvenCompactCiphertextList {
     }
 }
 
+pub struct ProvenCompactCiphertextListConformanceParams {
+    pub sk_params: ClassicPBSParameters,
+    pub elements_per_block: usize,
```

Comment:
Could this be called `elements_per_list/elements_per_vec/elements_per_sublist`, I think `block` may be too confusing with integer blocks